misc/open_pdks/condarc

```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```

Comment:
We want to avoid conda-forge as much as possible.

Reply:
I think that's required here because of the transitive dependencies of skywater-pdk.python (`dataclasses-json` and `typing_inspect`).

I was just trying to mimic what `misc/magic` does here: https://github.com/hdl/conda-eda/blob/master/misc/magic/condarc

Reply:
FYI With `defaults` as the top channel (e.g.: https://github.com/hdl/conda-eda/blob/master/pnr/nextpnr/fpga_interchange/condarc), `conda-forge` is used only for the packages unavailable in the `defaults` (main, msys2 and r) and `litex-hub` channels. Sometimes it results in building packages independent from `conda-forge` despite using it during building.

However, I can see this package has no dependencies at all after it's built so using `conda-forge` for all packages isn't a problem here.